src/components/Footer/components/FooterMedia/index.module.scss

```diff
@@ -36,6 +36,24 @@
     }
   }
 
+  .formWrap {
+    display: flex;
+    gap: 6px;
+
+    :focus-visible {
```

Comment:
This is valid for all children, and it is better to limit the scope to `&:focus-visible` or `> :focus-visible` .